lib/application.js

```diff
@@ -178,9 +178,7 @@ module.exports = class Application extends Emitter {
     if (this.silent) return;
 
     const msg = err.stack || err.toString();
-    console.error();
-    console.error(msg.replace(/^/gm, '  '));
-    console.error();
+    console.error(`\n${msg.replace(/^/gm, '  ')}\n`);
```

Comment:
How about `console.error('\n' + msg.replace(/^/gm, '  ') + '\n')`? I think it's a nice compromise.